### PR TITLE
Switch media-sound/pulseaudio to media-libs/libpulse

### PR DESCRIPTION
**Pull request by a17r**
Please also do unto them other ebuilds.

### CODE REVIEW

**Review by chewi — APPROVED** [review]

**Comment by thesamesam** [comment]
Let's try media-sound/pulseaudio in package.deprecated too

**Review by ZeroChaos- — APPROVED** [review]

**Review by floppym — APPROVED** [review]

**Review by ionenwks — APPROVED** [review]
games-emulation/dolphin: okay for this one, and anything else for games@
games-emulation/pcsxr: doesn't matter, it's last rited

**Comment by a17r** [comment]
> games-emulation/pcsxr: do what you want here or nothing, it's last rited and not getting saved

Ah right, did not check that. Thanks!

**Review by arthurzam — APPROVED** [review]

**Review by ajakk — APPROVED** [review]

File: net-wireless/blueman/blueman-9999.ebuild [hunk 1]
```diff
@@ -67,14 +67,11 @@ RDEPEND="
 			>=net-misc/networkmanager-0.8[introspection]
 		)
 	)
-	policykit? (
```
Comment (mgorny) [inline net-wireless/blueman/blueman-9999.ebuild:70]: How dare you change my perfect coding style?!

Comment (a17r) [reply net-wireless/blueman/blueman-9999.ebuild:70]: I try to leave a personal note in every code snippet I touch.

File: dev-python/pulsectl/pulsectl-23.5.2-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( pypy3 python3_{9..11} )
```
Comment (mgorny) [inline dev-python/pulsectl/pulsectl-23.5.2-r1.ebuild]: Please remove dead impls from the range when you touch any ebuilds with `PYTHON_COMPAT`, i.e. 9→10 here.

File: dev-python/pyglet/pyglet-2.0.7-r1.ebuild [hunk 3]
```diff
@@ -36,7 +36,7 @@ RDEPEND="
 	) )
 	sound? ( || (
 		media-libs/openal
-		media-sound/pulseaudio
+		media-libs/libpulse
```
Comment (mgorny) [inline dev-python/pyglet/pyglet-2.0.7-r1.ebuild:39]: Sort, please.

**Review by juippis — APPROVED** [review]

**Pull request merged**